testing/tests/DevExpress.ui.widgets.editors/datebox.mask.tests.js

```diff
@@ -811,6 +813,63 @@ module('Events', setupModule, () => {
 
         assert.ok(onInput.calledOnce);
     });
+
+    QUnit.test('click on the mask after clear value by clear button should not cause any errors, useMaskBehavior: true (T1094710)', function(assert) {
```

Comment:
I think we can si,plify it

```suggestion
    QUnit.test('click on input after clear button click should not cause any errors, useMaskBehavior: true (T1094710)', function(assert) {
```